### DISCUSSION TITLE
Antidos e non solo

### DISCUSSION THREAD

**Discussion by xrivo1** [rejetto/hfs#249] [Ideas]
Premesso che l'antidos funziona egregiamente, nel LOG access error sono presenti tantissime interrogazioni ( fallite ) di gente che non ha nient'altro da fare se non cercare di forzare il sistema.
Ritieni possibile creare una funzione che disincentivi tale comportamento?
Agendo magari proprio sulla lettura dei logs access error, bloccare automaticamente gli indirizzi IP che insistono nel loro intento, magari con le stesse funzioni dell'antidos ( numero tentativi e tempo )

**Comment by rejetto** [comment]
Hai già fatto questa cosa consigliata sul sito?
> have a domain (ddns is ok too), start vhosting plugin, configure your domain, enable "Block requests that are not using any of the domains above"

**Comment by xrivo1** [comment]
> Hai già fatto questa cosa consigliata sul sito?
> 
> > have a domain (ddns is ok too), start vhosting plugin, configure your domain, enable "Block requests that are not using any of the domains above"

yes and the accesses have decreased. apparently someone points directly to the domain name.

**Comment by xrivo1** [comment]
ok. scusa a volte scrivo italiano e a volte traduco :)

**Comment by xrivo1** [comment]
gli errori sono quasi sempre 404 o 401

**Comment by xrivo1** [comment]
ti allego un log di esempio
[access-error.log-2023-04-15.txt](https://github.com/rejetto/hfs/files/11365876/access-error.log-2023-04-15.txt)

**Comment by rejetto** [comment]
"disincentivi il comportamento" non vedo come. Il trucchetto di sopra
funziona tanto più quanto il tuo dominio non si trova "in giro" ma lo
conoscono solo in un circuito privato.
Potrei fare un filtro che ignora certe richieste, come quelle che finiscono
con .php

**Comment by xrivo1** [comment]
> "disincentivi il comportamento" non vedo come. Il trucchetto di sopra funziona tanto più quanto il tuo dominio non si trova "in giro" ma lo conoscono solo in un circuito privato. Potrei fare un filtro che ignora certe richieste, come quelle che finiscono con .php

Forse sarò troppo ossessivo :) ma credo che, tali tentativi siano fatti sempre dalla stessa ( o dalle stesse ) persone che utilizzano una VPN per cambiare indirizzo e conoscono il mio dominio.
Disincentivo il comportamento quando vedono che ogni volta, dopo 5, 6 , 10 tentativi vengo automaticamente bloccate. E che cavolo.... prima o poi si rompono di provare ahahahaha.  Che vadano da un'altra parte a perdere il loro tempo se non hanno nulla da fare.

**Comment by xrivo1** [comment]
è come quando blocchi i call center col cellulare. Una gran rottura di scatole. Blocca 20 numeri e non si fanno più sentire

**Comment by xrivo1** [comment]
> "disincentivi il comportamento" non vedo come. Il trucchetto di sopra funziona tanto più quanto il tuo dominio non si trova "in giro" ma lo conoscono solo in un circuito privato. Potrei fare un filtro che ignora certe richieste, come quelle che finiscono con .php

Questo è uno spunto interessante. ovviamente inutile fare un plugin solo per bannare le richieste che finiscono per php. magari potresti inserirlo nell'antidos. Ovviamente non sarebbe piu solo un antidos ma un anti richieste indesiderate..........
